### ISSUE TITLE
Enhancement: Support printers without z probe?

### ISSUE THREAD

**Issue by tablatronix** [npiegdon/bed-inspector#1]
Would it be possible to support printers without a zprobe, by using a secondary serial port to monitor for zstop commands or triggers, then I could use a microcontroller and a probe to get a map for example.

Or is this using something built into printers firmware to do the z travel and detect automatically?

**Comment by npiegdon** [comment]
You're correct that the printer firmware is handling it right now.  You just send a "G30" which makes the printer perform a measurement and dump the results back through the serial port.

It would take some rework (doing the firmware's job: Z travel in a loop while polling the sensor), but it's not outside the realm of possibility.  The whole utility is less than 300 lines (and written in a couple hours in the first place), so that kind of rework might not be so bad.
